jackson-jaxb/pom.xml

```diff
@@ -70,4 +70,24 @@
     </dependency>
   </dependencies>
 
+  <profiles>
+    <profile>
+      <activation>
+        <jdk>11</jdk>
+      </activation>
+
+      <dependencies>
+        <dependency>
+          <!--
+            JAXB was removed from java SDK on JEP 320
+            http://openjdk.java.net/jeps/320
+           -->
+          <groupId>org.glassfish.jaxb</groupId>
+          <artifactId>jaxb-runtime</artifactId>
+          <version>2.4.0-b180830.0438</version>
```

Comment:
Should we use this dependency or the general `jaxb` one?

```
<!-- https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api -->
<dependency>
    <groupId>javax.xml.bind</groupId>
    <artifactId>jaxb-api</artifactId>
    <version>2.3.1</version>
</dependency>
```

Reply:
Done'ish...

still need the runtime to run tests... but dependencies are a bit better now